tools/pyi/gen_pyi.py

```diff
@@ -1319,17 +1311,18 @@ def replace_special_case(hint: str) -> str:
                 )
             ],
             "__contains__": [defs("__contains__", ["self", "item: Any", "/"], "_bool")],
-            "__getitem__": [defs("__getitem__", ["self", INDICES, "/"], "Tensor")],
+            "__getitem__": [defs("__getitem__", ["self", INDICES, "/"], "Self")],
             "__setitem__": [
                 defs(
                     "__setitem__",
-                    ["self", INDICES, "value: Tensor | Number", "/"],
+                    ["self", INDICES, "value: Tensor | Number | _complex", "/"],
                     "None",
                 )
             ],
-            "tolist": [defs("tolist", ["self"], "list")],
+            "tolist": [
+                defs("tolist", ["self"], "NestedList")],
```

Comment:
Driveby as currently annotated as `list` (which is incomplete according to `pyright`)